xml/System.Net.Http/HttpClient.xml

```diff
@@ -1557,7 +1604,15 @@ The <paramref name="requestUri" /> is not an absolute URI.
  ]]></format>
         </remarks>
         <exception cref="T:System.InvalidOperationException">The <paramref name="requestUri" /> must be an absolute URI or <see cref="P:System.Net.Http.HttpClient.BaseAddress" /> must be set.</exception>
-        <exception cref="T:System.Net.Http.HttpRequestException">The request failed due to an underlying issue such as network connectivity, DNS failure, server certificate validation or timeout.</exception>
+        <exception cref="T:System.Net.Http.HttpRequestException">The request failed due to an issue getting a valid HTTP response such as network connectivity failure, DNS failure, server certificate validation error or invalid server response. On .NET 8+ the reason is indicated by <xref:System.Net.Http.HttpRequestException.HttpRequestError?displayProperty=nameWithType>
+
+-or-
+
+the respose status code was outside of the range of 200-299 (status codes indicating success according to the standard)
```

Comment:
I'm confused, I thought that we're not supposed to document stored exception as a regular exception?